src/mito2/src/metrics.rs

```diff
@@ -437,6 +437,12 @@ lazy_static! {
             "mito stalled write request in each worker",
             &[WORKER_LABEL]
         ).unwrap();
+    /// Number of ref files per table
+    pub static ref GC_TABLE_REF_FILE_CNT: IntGaugeVec = register_int_gauge_vec!(
+            "greptime_gc_table_ref_file_count",
+            "gc table ref file count",
+            &["table_id"]
```

Comment:
Will it be a problem if there are too many tables?

Reply:
> Will it be a problem if there are too many tables?

probably, then only store one number or what?

Reply:
I prefer to store only one number.